EIPS/eip-5380.md

```diff
@@ -0,0 +1,181 @@
+---
+eip: 5380
+title: EIP-4907 Alternative Design
+description: Add a time-limited role with restricted permissions to EIP-721 tokens.
+author: Anders (@0xanders), Lance (@LanceSnow), Shrug <shrug@emojidao.org>, Tim Daubenschütz (@TimDaub)
+discussions-to: https://ethereum-magicians.org/t/idea-erc-721-user-and-expires-extension/8572
```

Comment:
EIPs should generally have their own discussion link to avoid confusion.  I recommend linking to this in the OP of that discussion topic, but we should keep the topics separate so there isn't confusion about what EIP is being discussed.

Reply:
```
discussions-to: https://ethereum-magicians.org/t/pr-5380-eip-4907-alternative-design/10190
```